hbase-server/src/main/java/org/apache/hadoop/hbase/master/MasterRpcServices.java

```diff
@@ -3289,4 +3295,24 @@ public UpdateRSGroupConfigResponse updateRSGroupConfig(RpcController controller,
     }
     return builder.build();
   }
+
+  @Override
+  public MasterProtos.BalancerDecisionsResponse getBalancerDecisions(RpcController controller,
```

Comment:
This should implement getLogEntries() instead, or be removed if getLogEntries is already implemented elsewhere. 

Use getLogEntries() to retrieve balancer decisions

Reply:
At RPC level, we can't achieve abstraction since protobuf doesn't support `is-a` relationship among proto `messages`.